x/ecocredit/spec/01_concepts.md

```diff
@@ -11,6 +11,17 @@ A credit class is the primary abstraction for ecosystem service credits and is d
 
 Once a new credit class is created, credits can be issued at-will in distinct batches by any address in the **issuer list** of the corresponding credit class.
 
+## Project
+
+A project describes the high-level on-chain information for an issuer associated with a credit class. A project is defined by:
+- **project ID**: The unique ID of the project
+- **class ID**: The unique ID of the credit class for this project
+- **issuer***: The issuer of the credit batches for this project
+- **project location**: The location of the project backing the credits issued from this project
+- **metadata**: A byte array (up to 256 bytes) which can be used to store small amounts of metadata, or a URI that points to an off-chain resource for querying more complete metadata information. This usually would include descriptive information about the project.
+
+Each credit batch is associated with a project, backing each issuance with information about the issuer of the credits. A credit class can have multiple projects that issue credits from the credit class. It is not required that the same issuer issue all batches for a project.
```

Comment:
"It is not required that the same issuer issue all batches for a project."
what do you mean by that? looking at the code, it seems like the project issuer should be the one issuing all batches for this project.

Reply:
Hmm... I'm not entirely sure what I was trying to say. Maybe it is not required that the same "project" issue all batches for a "credit class" but that would be redundant following the sentence before. I'll remove.